src/objc/TuriCreateObjC.m

```diff
@@ -25,6 +25,7 @@ static bool handleError(tc_error* error_ptr, NSError** error) {
                                          code:MLModelErrorGeneric
                                      userInfo:userInfo];
         }
+        assert(false);
```

Comment:
Do we need the `assert(false)`?

Reply:
Probably not (see also Toby's comment). I'll remove it.